refactor(server): Update system emails

andrewwallacespeckle opened the pull request:
Email Verification:
<img width="614" alt="image" src="https://github.com/user-attachments/assets/b46a2ff1-7c67-452a-a007-68ee066d0163" />

Workspace request:
<img width="559" alt="image" src="https://github.com/user-attachments/assets/9f47714c-8287-460f-b9f7-2a744ddffaa5" />

Workspace request approved:
<img width="567" alt="image" src="https://github.com/user-attachments/assets/0561b726-c002-4519-81ff-a76993380914" />

Workspace request denied:
<img width="615" alt="image" src="https://github.com/user-attachments/assets/ff9940cf-0a58-4d01-91bf-ed74f6949ab6" />

Mentioned in comment:
<img width="508" alt="image" src="https://github.com/user-attachments/assets/6d9113ab-e33b-4680-acc4-256f13430cc3" />

Invited to project:
<img width="549" alt="image" src="https://github.com/user-attachments/assets/95db9cbf-5357-4063-86a6-d8141ac0da43" />

Password reset:
<img width="516" alt="image" src="https://github.com/user-attachments/assets/9f74b0b5-4fd2-488f-9089-8af56d36bd5d" />

benjaminvo commented:
Very nice Andrew!

- I think the logo in the top could be smaller across all emails
- How does the logo work in dark mode email clients? If the "Speckle" part is a problem then we could also just show the blue cube without company name.
- What's the font in your screenshots? :) Does that come from your system or from the font we've defined in the emails?

michalspeckle commented:
Great improvement!

- Yes, the cube as a logo sounds like a good idea.
- There are small typos/missing spaces between works here and there in the screenshots you shared but I can't find them in the code.

andrewwallacespeckle commented:
> * There are small typos/missing spaces between words here and there in the screenshots you shared but I can't find them in the code.

I noticed some of these after I had done these screenshots and fixed.

andrewwallacespeckle commented:
> * What's the font in your screenshots? :) Does that come from your system or from the font we've defined in the emails?

We use MailJet to build the emails, it builds them in a way that is best supported by all clients. 

It seems to set the default font as:
Ubuntu, Helvetica, Arial, sans-serif

We can override this, but we'd still need to use fonts that users would have installed on their device.

michalspeckle commented:
> @benjaminvo @michalspeckle How small do you want to go? I like 40. I can adjust the spacing a bit once we decide on the size of the logo.

I like 40 as well. 

Btw: I've  just checked which email clients support @media prefers-color-scheme (to see what stops us from swapping the image in dark mode) and I'm shocked that Gmail offers absolutely none => https://www.caniemail.com/search/?s=prefers-color-scheme.

michalspeckle commented:
...wow, font-face support is no different here https://www.caniemail.com/features/css-at-font-face/ 

In this case I'd prefer using Helvetica, Arial as a default font.

Mikehrn commented:
> ...wow, font-face support is no different here https://www.caniemail.com/features/css-at-font-face/
> 
> In this case I'd prefer using Helvetica, Arial as a default font.

The better question is "What DO emails support"? There is a reason emails are still built with tables 😅

andrewwallacespeckle commented:
<img width="555" alt="image" src="https://github.com/user-attachments/assets/23fe882d-ba30-448b-8ca3-826f1780e9aa" />
- Logo set to 40px
- Adjusted spacing around logo
- Updated to Helvetica, Arial

benjaminvo commented:
<!-- linear:replyTo:2700997268 -->

Looks good!

michalspeckle commented:
Looks nice! Nit: we have unnecessary extra spaces in the footer here (before the coma and the full stop)

<img width="179" alt="Screenshot 2025-03-06 at 19 05 48" src="https://github.com/user-attachments/assets/3210a426-1112-403b-a7b7-9e6a64037ca9" />

michalspeckle reviewed (dismissed):
LGTM
